### ISSUE TITLE
[FEATURE] Overlay for tab-panels

### ISSUE THREAD

**Issue by andcarpi** [ColorlibHQ/AdminLTE#2609]
**Is your feature request related to a problem? Please describe.**
Tried to use 
```
<div class="overlay dark">
       <i class="fas fa-2x fa-sync-alt fa-spin"></i>
</div>
```
in a tab-panel, but didn't worked, it pushes content instead of overlaying it

**Describe the solution you'd like**
The overlay should act the same way in widgets or cards

**Extra**
It would be nice to have the option to add text over or under the icon. :)

![image](https://user-images.githubusercontent.com/13441557/77278481-37802900-6c9e-11ea-967f-3117674a5539.png)

**Comment by andcarpi** [comment]
Managed to achieve the result... woulda be nice to have them available as classes

**tab-pane div open**
```
<div class="overlay-wrapper" style="position:relative">
    <div class="overlay" style="flex-direction: column">
        <i class="fas fa-3x fa-sync-alt"></i>
        <div>Carregando</div>
    </div>
    Tab-panel content here
</div>
```
**tab-pane div close**

**Comment by REJack** [comment]
Feature is released with the latest version (v3.0.3), thanks for your help with that @andcarpi.
